### PR TITLE
[CI] run benchmarks on PR

### PR DESCRIPTION
**Pull request by apbard**

### CODE REVIEW

File: .github/workflows/benchmarks_pr.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+name: Benchmarks
```
Comment (vmoens) [inline .github/workflows/benchmarks_pr.yml:1]: Maybe Continuous Benchmarks to match the other workflow?

Comment (apbard) [reply .github/workflows/benchmarks_pr.yml]: I thought they would conflict

Comment (vmoens) [reply .github/workflows/benchmarks_pr.yml]: no you will just see the branch appear (main or the one from the PR)

File: .github/workflows/benchmarks_pr.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+name: Continuous Benchmarks
```
Comment (vmoens) [inline .github/workflows/benchmarks_pr.yml:1]:
```suggestion
name: Continuous Benchmark
```

**Comment by vmoens** [comment]
They are duplicated now haha

I guess they should be in the same file, is it possible?

**Comment by apbard** [comment]
> They are duplicated now haha
> 
> I guess they should be in the same file, is it possible?

let's start with getting this to work... :)

**Pull request merged**